framework/wazuh/rbac/decorators.py

```diff
@@ -235,6 +238,7 @@ def _get_required_permissions(actions: list = None, resources: list = None, **kw
                 target_params[m.group(1)] = m.group(2)
             else:  # Static resource
                 target_params[m.group(1)] = '*'
+                add_denied = not broadcast.get()
```

Comment:
This line should moved outside the if-else block.

Reply:
Done